app/main.py

```diff
@@ -1 +1,38 @@
-# write your code here
+from __future__ import annotations
+
+
+class Animal:
+    alive = []
+
+    def __init__(
+        self, name: str, health: int = 100, hidden: bool = False
```

Comment:
It is a good practice to place all arguments in the `__init__` on the new line to improve the type annotations' readability. Follow this approach in the future.